### PR TITLE
Api 11291 updated test container

### PR DESCRIPTION
**Pull request by esubocz-va**
Related story: https://vajira.max.gov/browse/API-11291

See README.md for testing instructions.
QA test vars on this branch are those that should be decrypted and used locally: https://github.com/department-of-veterans-affairs/lighthouse-veteran-verification-deployment/pull/36/files

Expected results of local run (hitting vets-api VV): 
1 failure expected as `/backend/health` endpoint is Java VV specific.

```
.
+-- JUnit Jupiter [OK]
| +-- ServiceHistoryOauthIT [OK]
| | +-- serviceHistoryNotFound() 23171 ms [OK]
| | +-- serviceHistoryBadScopes() 21566 ms [OK]
| | '-- serviceHistoryOauthHappyPaths() 17645 ms [OK]
| +-- OpenApiIT [OK]
| | '-- veteranVerificationDocs() [OK]
| +-- DisabilityRatingOauthIT [OK]
| | +-- disabilityRatingOauthHappyPaths() 18589 ms [OK]
| | +-- noBgsUser() 21327 ms [OK]
| | '-- disabilityRatingBadScopes() 20401 ms [OK]
| +-- DisabilityRatingIT [A] assumption was not met due to: Skipping in QA
| +-- BackendHealthIT [OK]
| | '-- HappyPath() [X] 1 expectation failed.
| |       Expected status code <200> but was <404>.
| +-- ServiceHistoryIT [A] assumption was not met due to: Skipping in QA
| +-- VeteranStatusIT [A] assumption was not met due to: Skipping in QA
| +-- KeysIT [OK]
| | '-- happyPath() [OK]
| '-- VeteranStatusOauthIT [OK]
|   +-- veteranStatusNotConfirmed() 19234 ms [OK]
|   +-- veteranStatusOauthHappyPaths() 19591 ms [OK]
|   +-- veteranStatusV5OauthHappyPaths() 20380 ms [OK]
|   '-- veteranStatusBadScopes() 19290 ms [OK]
'-- JUnit Vintage [OK]```

### CODE REVIEW

File: veteran-verification/src/main/java/gov/va/api/lighthouse/veteranverification/service/controller/servicehistory/VeteranServiceHistoryController.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ public ServiceHistoryResponse veteranServiceHistoryResponse(
     try {
       deployments = emisClient.deploymentRequest(ediPiOrIcn);
     } catch (ServerSOAPFaultException exception) {
-      throw new NoServiceHistoryFoundException();
+      deployments = null;
```
Comment (beaugrantham-va) [inline veteran-verification/src/main/java/gov/va/api/lighthouse/veteranverification/service/controller/servicehistory/VeteranServiceHistoryController.java:69]: How does this relate to the test container? And is the behavior correct? I'd expect a SOAP fault to result in a server error.

Comment (esubocz-va) [reply veteran-verification/src/main/java/gov/va/api/lighthouse/veteranverification/service/controller/servicehistory/VeteranServiceHistoryController.java]: this was residual debugging stuff, went unchecked due to bad test. reverted change, and updated test. good catch.

File: veteran-verification-tests/src/test/java/gov/va/api/lighthouse/veteranverification/tests/VeteranStatusOauthIT.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import gov.va.api.lighthouse.veteranverification.api.v1.VeteranStatusResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+@Slf4j
+public class VeteranStatusOauthIT {
+  @BeforeAll
+  static void assumeEnvironment() {
+    // Tests are only ran in environments that are blocked by the kong gateway (excluding
```
Comment (dbrown-va) [inline veteran-verification-tests/src/test/java/gov/va/api/lighthouse/veteranverification/tests/VeteranStatusOauthIT.java:25]: I'm not sure if `blocked` is the right wording, vs something like `fronted` or `routed through`.

Or perhaps slightly more detail like `that have the kong gateway providing OAuth token and scope validation)`.

**Review by dbrown-va — APPROVED** [review]
Really like the simple way of running locally vs against a deployed instance like dev-api.

This will help a lot as we get to the point of migrating to the java upgrade version.

File: veteran-verification-tests/src/test/java/gov/va/api/lighthouse/veteranverification/tests/VeteranStatusOauthIT.java (hunk 2 above)
Comment (esubocz-va) [reply veteran-verification-tests/src/test/java/gov/va/api/lighthouse/veteranverification/tests/VeteranStatusOauthIT.java]: I fix 👍

**Review by dbrown-va — APPROVED** [review]

**Pull request merged**